### ISSUE TITLE
INA260 Power Sensor not getting configured

### ISSUE THREAD

**Issue by CFD2** [ccxtechnologies/nuttx#9]
### Summary
Writing to configuration register of the power sensor and reading from it results in default (a.k.a. reset) value 0x6127. Can write to another register of the power sensor (Alert Limit Register @ 0x07) just fine.
### Description
Sensor address: 0x40
Configuration register address: 0x00
Default register value: 0x6127

Added a debug statement to read from config register right after writing to it:
> imxrt_ina260initialize: Initializing INA260
> ina260_access: start_register_address: 0x00 data_length: 2 register_value: 0x02 (0x0002) ret: 0
> ina260_access: Reading back config register:
> ina260_access: start_register_address: 0x00 data_length: 2 register_value: 0x61 (0x2761) ret: 0
> ina260_register: (addr=0x40) registered at /dev/ina260
<img width="1390" alt="INA260 writing config register crop" src="https://user-images.githubusercontent.com/5952749/125665482-9e081a51-9c51-4665-99e3-7f41f796f158.png">

**Comment by CFD2** [comment]
From [datasheet, p.21](https://www.ti.com/lit/ds/symlink/ina260.pdf):
> The Configuration Register can be read from at any time without impacting or affecting the device settings or a
conversion in progress. Writing to the Configuration Register halts any conversion in progress until the write
sequence is completed resulting in a new conversion starting based on the new contents of the Configuration
Register (00h).

The register is available for read/write operations at any time.

**Comment by CFD2** [comment]
On the screenshot in the header message, the format of I2C message is incorrect for writing to registers. It is similar to reading from the register (i.e. change the register pointer, then supply data MSB and LSB, figure 26 on the screenshot from the datasheet).

Instead, when writing to a register, the address needs to be followed by the destination register location and then followed by the data to be written into the register.

From the datasheet, p.19:
<img width="1066" alt="Screenshot_20210715_121400" src="https://user-images.githubusercontent.com/5952749/125821681-60b9c2c9-681d-4d7e-aad3-9cffc5177812.png">

In other words, writing should look more like this (addr. 40, reg. 00, data 02 07):
<img width="1530" alt="ina260 reg write proper" src="https://user-images.githubusercontent.com/5952749/125822211-4d155ae4-33f7-4fc2-85ef-7163a2afdc2e.png">
